src/templates/proj_layout.html

```diff
@@ -3,14 +3,18 @@
     <head>
         <meta charset="UTF-8">
         <title>{% block title %}ICCM BOF app{% endblock %}</title>
-        <link rel="stylesheet" href="/assets/css/main.css">
-	    <meta http-equiv="refresh" content="5" />
+        <link rel="stylesheet" href="/assets/css/{{branding}}.css">
+	    <meta http-equiv="refresh" content="500" />
```

Comment:
500 seconds for the refresh? that is more than 8 minutes.
I agree 5 seconds is perhaps too frequent. Maybe change to 60 seconds?